schema/validator.go

```diff
@@ -38,7 +41,7 @@ func (e ValidationError) Error() string {
 }
 
 // Validate validates the given reader against the schema of the wrapped media type.
-func (v Validator) Validate(src io.Reader) error {
+func (v Validator) Validate(src io.Reader, verbose bool, out *log.Logger) error {
```

Comment:
@stevvooe  Agree. I find that current validate functions are all use appended argument of `log.Logger`, so I keep the style. Should I fix also all of them too? 
https://github.com/opencontainers/image-spec/blob/master/image/image.go#L30
https://github.com/opencontainers/image-spec/blob/master/image/image.go#L37
https://github.com/opencontainers/image-spec/blob/master/image/image.go#L52